pkg/client/sql.go

```diff
@@ -45,6 +47,7 @@ func (c *immuClient) SQLExec(ctx context.Context, sql string, params map[string]
 	return c.ServiceClient.SQLExec(ctx, &schema.SQLExecRequest{Sql: sql, Params: namedParams})
 }
 
+// SQLQuery performs a query (read-only) operation.
```

Comment:
it would be good to add some explanations of params in general. I.e. renewSnapshot meaning is not so clear for the users I believe.

Reply:
Added descriptions for some parameters in f76da91e1ed62ff475e23e8a3ccd7a47b3bd19f0 (the mentioned `renewSnapshot` was a good example since it is no longer used and would then be confusing).